File: Arrays and Hashing/217-Contains-Duplicate/217.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> numSet = new HashSet<int>();
        foreach(var num in nums){
            if(numSet.Contains(num)){
                return true;
            }
            else{
                numSet.Add(num);
            }
        }
        return false;
    }
}

File: Arrays and Hashing/217-Contains-Duplicate/217.cpp
#include <unordered_set>

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        std::unordered_set<int> numSet;
        for(int i = 0; i < nums.size(); i++){
            if (numSet.contains(nums[i])){
                return true;
            }
            else{
                numSet.insert(nums[i]);
            }
        }
        return false;
    }
};